regression-tests/18-compile-arm-ports/Makefile

```diff
@@ -54,6 +54,8 @@ sensniff/ev-aducrf101mkxz \
 cfs-coffee/cc2538dk \
 cfs-coffee/openmote-cc2538 \
 cfs-coffee/zoul \
+fat/zoul:BOARD=remote-reva \
```

Comment:
Yup, bypassing the MOSFET would be required to properly power the MicroSD, I didn't remembered that.